src/components/pages/HomePage/index.jsx

```diff
@@ -28,7 +29,9 @@ import Projects from "../../../assets/content/projects.jpg";
 
```

Comment:
```suggestion
```

Reply:
We split imports per header and a new line.